src/components/SchemaTable/index.css

```diff
@@ -0,0 +1,20 @@
+* {

```

Comment:
'*' can affect performance. If we can avoid using this, it'd be good.

Reply:
Removed the entire stylesheet since utilizing material-ui's css-in-js methods!